Node-DC-EIS-client/runspec.py

```diff
@@ -659,41 +659,30 @@ def print_meminfo():
   elapsed_time = time.time() - start_time
   with open(os.path.join(os.path.join(results_dir,directory),directory+"-"+memlogfile+".csv"), 'wb') as f:
     writer = csv.writer(f)
-    writer.writerows(izip(list(range(0, int(elapsed_time), 1)),rss_list,heapTotlist,heapUsedlist))    
+    writer.writerows(izip(list(range(0, int(elapsed_time), 1)),rss_list,heapTotlist,heapUsedlist))
 
 #send concurrent requests to the server using the urls generated 
 # decides the type of the url depending on the method GET, POST or DELETE.
-def send_request(employee_idlist):
-  global after_run
-  global phase
+def requestBasedRun(pool, log, employee_idlist):
```

Comment:
The changes are kind of hard to see due to the way diff works. It might be better to add this function after send_request(). Python doesn't care about the order in which they are declared.